glusterd2/main.go

```diff
@@ -25,6 +25,9 @@ import (
 	config "github.com/spf13/viper"
 	"github.com/thejerf/suture"
 	"golang.org/x/sys/unix"
+
```

Comment:
I guess this extra line is not required since these are 3rd party libraries

Reply:
Noted. Will remove it.